frontend/vue/components/UserAccount/PrivacySection.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <section class="privacy-section">
+    <h1 class="privacy-section__title">
+      Privacy
+    </h1>
+    <h4 class="privacy-section__subtitle privacy-section__subtitle_general">
+      IBM Privacy Statement
+    </h4>
+    <AppLink class="privacy-section__policy-link" url="https://www.ibm.com/privacy">
+      https://www.ibm.com/privacy
+    </AppLink>
+    <h4 class="privacy-section__subtitle privacy-section__subtitle_quantum">
+      IBM Quantum End User Agreement
```

Comment:
same comment, we should add translation here and define the string in `strings.yaml`